pd/src/components/staking.rs

```diff
@@ -280,7 +285,7 @@ impl Component for Staking {
 ///
 /// TODO: should this be split into Read and Write traits?
 #[async_trait]
-pub trait View: WriteOverlayExt + Send + Sync {
+pub trait View: WriteOverlayExt + Send + Sync + Sized {
```

Comment:
Hmm, maybe we should move the `Send + Sync + Sized` bounds to the `WriteOverlayExt` trait, so that they're implied for every `View`?

Reply:
That sounds like a good plan